pallets/admin-utils/src/lib.rs

```diff
@@ -624,6 +710,9 @@ pub mod pallet {
             Ok(())
         }
 
+        /// The extrinsic sets the network rate limit for the network.
+        /// It is only callable by the root account.
+        /// The extrinsic will call the Subtensor pallet to set the network rate limit.
```

Comment:
Some docstrings say "for the network" and others "for the subnet". We should clarify how "for the network" differs from "for the subnet", and also clarify "for the network" if root account still synonymous with the subnet owner?

Reply:
how about we use "for the network" if there is no netuid in the parameter. and "for the subnet" with specific netuid.

Reply:
But first, what does "for the network" mean here? What is it referring to? I am not getting the meaning of "network rate limit for the network" here. Also it now says "callable by the root account"... I am not getting this either. Root account is doing what here? When we know that, we can tweak the wording then.

Reply:
the network is subtensor blockchain in the context. I think subnet is separate network, they are connected by its own connection method. but sutnet's validators are also playing important role in subtensor blockchain. 
I think we need the terminology for our blockchain, the whole system including chain and all subnets.
The root account is a special account in substrate based chain, it can call any function and we can set the different privilege for normal account and root account. At the beginning of blockchain launch, it is a special real account. Later, it is from governance.

Reply:
yeah, often "network" is used to refer to the entire blockchain network collectively, so "all of finney" for example